internal/host/host.go

```diff
@@ -1275,3 +1278,36 @@ func (m *Manager) GetHostByKubeKey(key types.NamespacedName) (*common.Host, erro
 func (m *Manager) UnRegisterHost(ctx context.Context, hostID, infraEnvID string) error {
 	return common.DeleteHostFromDB(m.db, hostID, infraEnvID)
 }
+
+func (m *Manager) GetReadyAndApprovedHostsSummary(clusterID strfmt.UUID) (registered, approved int, err error) {
```

Comment:
Would a better name be `GetKnownHostApprovedCount` or something?

Specifically the `Ready` word confused me. Not sure why we wouldn't just use the actual status name.

Reply:
I think "ready" is more suitable than "known".
Will change to GetKnownHostApprovedCounts